backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties

```diff
@@ -1,24 +1,47 @@
-info=Bitte grenzen Sie ihre Auswahl weiter ein so das max. 150.000 Datens�tze ausgegeben werden. Aktuell wurden aktuelle
-info2=Datens�tze selektiert.
-
-title=Konten - Information
-accountno=Konto-Nr.
-account=Konto
-dateacct=Buchungsdatum
-activity=Kostenstelle
-record_id=Datensatz ID
-bp_name=Gesch�ftspartner
-description=Beschreibung
-counter_account=Gegenkonto
-amtdr=Soll
-amtcr=Haben
-balance=Saldo
-docstatus=Belegstatus
-
-page=Seite
-of=von
-all=Alle
-
-initial_balance=Anfangssaldo
-sum=Summe
-sum_eur=Summe in EUR
\ No newline at end of file
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+account = Konto
+
+accountno = Konto-Nr.
+
```

Comment:
What's this?
Why did you modify it?

Reply:
@cristinamghita  
this was generated/reordrerd by : `#Generated by Eclipse Messages Editor (Eclipse Babel)`

Reply:
fixed